api/tests/auxilary.test.ts

```diff
@@ -40,10 +40,10 @@ test('Env: jwt expiration in seconds format', async () => {
 });
 
 describe('Healthz', () => {
-  vi.mock('../src/database');
-
   test('healthz: returns 200', async () => {
-    prisma.user.findFirst.mockResolvedValueOnce({ id: '1', username: 'asdf', password: 'asdf' });
```

Comment:
this was mocked before. Mock removed because pothos pluging expects a prisma object as part of it config. Hence, in case prisma gets mocked, it breaks the pothos plugin and everything crashes.

> (I like gql stack.)